covid-vue-app/test/frontend-test.js

```diff
@@ -0,0 +1,189 @@
+const {Builder, By} = require('selenium-webdriver');
+const assert = require('assert');
+
+const driver = new Builder().forBrowser("firefox").build();
+var numberOfTestsCompleted = 0;
+var numberOfErrors = 0;
+const numberOfTests = 5;
+
+
+async function openTheApp() {
+    try{
+        await driver.get("http://192.168.1.11:8080/");
```

Comment:
Change to `localhost` (127.0.0.1), not sure if this would work :L

Reply:
Sure thing!